### PR TITLE
account for duplicates

### PR DESCRIPTION
**Pull request by bryceVeraSolutions**
assign multiple tags to single contact

### CODE REVIEW

**Review by mtuchi — CHANGES_REQUESTED** [review]
@bryceVeraSolutions the changes you proposed does not work
have you run this workflow locally to confirm that it will give you the desired output?

File: jobs/getCampaignMembers.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ fn(state => {
 
   for (const member of campaignMembers) {
     const mappedMember = {
-      subscriber_hash: member.Email,
+      //subscriber_hash: member.Email,
```
[the remark this answers is not in the document]
Comment (bryceVeraSolutions) [reply jobs/getCampaignMembers.js:34]: Yes this is working

**Comment by bryceVeraSolutions** [comment]
@mtuchi updated the pull request, ready for review

File: jobs/getCampaignMembers.js (hunk 1 above)
Comment (mtuchi) [inline jobs/getCampaignMembers.js:34]: @bryceVeraSolutions please remove this comment and share your feedback with Willian why we don't need this

File: jobs/getCampaignMembers.js [hunk 2]
```diff
@@ -49,10 +49,57 @@ fn(state => {
...
+
+      if (existingItem) {
+        existingItem.tags = [...new Set([...existingItem.tags, ...item.tags])];
+      } else {
+        result.push(item);
+        console.log('here is the result' + result);
+      }
+      return result;
+    }, []);
+    console.log(
```
Comment (mtuchi) [inline jobs/getCampaignMembers.js:90]: @bryceVeraSolutions please remove this `console.log()`. It's not important on PROD

File: jobs/getCampaignMembers.js [hunk 3]
```diff
@@ -49,10 +49,57 @@ fn(state => {
...
+      if (existingItem) {
+        existingItem.tags = [...new Set([...existingItem.tags, ...item.tags])];
+      } else {
+        result.push(item);
+        console.log('here is the result' + result);
+      }
+
+      return result;
+    }, []);
+    console.log(
```
Comment (mtuchi) [inline jobs/getCampaignMembers.js:70]: @bryceVeraSolutions please remove this console.log

**Review by mtuchi — CHANGES_REQUESTED** [review]
Hiya @bryceVeraSolutions please work on the changes i have requested, Once those are done you can go ahead and merge this PR and share an update on slack for Willian to proceed testing the workflow

File: jobs/getCampaignMembers.js (hunk 1 above)
Comment (bryceVeraSolutions) [reply jobs/getCampaignMembers.js]: made the updates

File: jobs/getCampaignMembers.js (hunk 3 above)
Comment (bryceVeraSolutions) [reply jobs/getCampaignMembers.js]: removed it

File: jobs/getCampaignMembers.js (hunk 2 above)
Comment (bryceVeraSolutions) [reply jobs/getCampaignMembers.js]: removed it

**Comment by bryceVeraSolutions** [comment]
@mtuchi I made all the requested updates and am merging to main

**Pull request merged**